lib/services/adapter/index.ts

```diff
@@ -22,6 +22,8 @@ class AdapterManager extends AbstractManager {
 
       // set transformed context
       input.attributesManager.setPersistentAttributes(transformedContext);
+
+      log.warn('transformed context (skillId, userId)', versionID, input.requestEnvelope?.context?.System?.user?.userId);
```

Comment:
https://github.com/alexa/alexa-skills-kit-sdk-for-nodejs/blob/2.0.x/ask-sdk-dynamodb-persistence-adapter/lib/attributes/persistence/PartitionKeyGenerators.ts#L31